src/yuzu/configuration/configure_network.cpp

```diff
@@ -26,7 +26,14 @@ void ConfigureNetwork::ApplyConfiguration() {
     Settings::values.network_interface = ui->network_interface->currentText().toStdString();
 }
 
-void ConfigureNetwork::RetranslateUi() {
+void ConfigureNetwork::changeEvent(QEvent* event) {
+    if (event->type() == QEvent::LanguageChange) {
+        RetranslateUI();
+    }
+    QWidget::changeEvent(event);
```

Comment:
```suggestion
    }
    
    QWidget::changeEvent(event);
```
For consistency with other files